src/admin/containers/member_page.js

```diff
@@ -21,7 +21,7 @@ const ViewMember = React.createClass({
     fetch_member(id)
   },
 
-  deactivate_member_click (_) {
+  deactivate_member_click () {
```

Comment:
It was a linting error. Hadn't though that it provided some information about the function actually, will put back it.